swift/src/CompileSlicePlugin/CompileSlicePlugin.swift

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) ZeroC, Inc. All rights reserved.
+
+import Foundation
+import PackagePlugin
+
+enum PluginError: Error {
+    case invalidTarget(Target)
+    case missingCompiler(String)
+    case missingConfigFile(String, String)
+    case missingIceSliceFiles(String)
+
+    var description: String {
+        switch self {
+        case .invalidTarget(let target):
+            return "Expected a SwiftSourceModuleTarget but got '\(type(of: target))'."
```

Comment:
I believe you can simplify and remove "return"

Reply:
Compiler doesn't like it.